roles/advanced-core/advanced_dhcp_server/templates/dhcpd.subnet.conf.j2

```diff
@@ -32,14 +32,15 @@
       {% set ipxe_rom_filename = (ipxe_architecture + '/standard_' + ipxe_driver + 'ipxe.efi') %}
     {% else %}  {# Assume everything else is pcbios #}
       {% set ipxe_rom_filename = (ipxe_architecture + '/standard_undionly.kpxe') %}
-    {% endif %}
+    {% endif -%}
 
-    {% for nic, nic_args in hostvars[host]['network_interfaces'].items() %}
+    {%- for nic, nic_args in hostvars[host]['network_interfaces'].items() %}
```

Comment:
The minus sign should remove whitespace before the block here, it is redundant with `lstrip_blocks: "True"` (1st line of the file)